internal/forms/forms.go

```diff
@@ -704,6 +705,49 @@ func (m *Manager) findAbsPathForTemplatePath(tmplPath string) (string, error) {
 	return retVal, nil
 }
 
+func gpsPos(decimal bool, signed bool) string {
+	// return current GPS position in one of these alternative formats:
+	// decimal=false:              degrees-minutes.seconds w/Northing and Easting
+	// decimal=true, signed=false: degrees.fraction w/Northing and Easting
+	// decimal=true, signed=true:  degrees.fraction w/minus sign for South and West
+
+	// documentation: https://www.winlink.org/sites/default/files/RMSE_FORMS/insertion_tags.zip
+	// <GPS> - Inserts the current GPS location (if available)
+	// Ex: 46-22.77N 121-35.01W
+	// <GPS_DECIMAL> - Inserts the current GPS location (if available)
+	// Ex: 46-22.77N 121-35.01W (this example is probably incorrect)
+	// Ex: 46.3795N 121.5835W
+	// <GPS_SIGNED_DECIMAL> - Inserts the current GPS location as a signed decimal latitude/longitude (if available)
+	// Ex: 46.3795 -121.5835
+
+
+	addr := "localhost:2947" // TODO: get from config
```

Comment:
Within the Manager struct (most of the functions here are actually methods on that struct), the config object is available. So as long as that's in scope, getting this value should be as simple as m.config.GPSd.Addr.

Reply:
Actually I tried this at first and there's a local version of Config that appears not to have GPSd.... 

https://github.com/la5nta/pat/blob/ccb493474debb2a84ee2ab60b292327426c86f8a/internal/forms/forms.go#L55-L62

Reply:
Ah, I see you're right. In that case you'll need to add a new field to forms.Config to hold the GPSd address, and plumb it though in main.go where we call forms.NewManager.